lib/token.ts

```diff
@@ -671,11 +671,9 @@ function addOAuthParamsToStorage(sdk: OktaAuth, tokenParams: TokenParams, urls)
     urls,
     ignoreSignature
   });
-  if (browserStorage.browserHasSessionStorage()) {
-    browserStorage.getSessionStorage().setItem(REDIRECT_OAUTH_PARAMS_NAME, tokenParamsStr);
-  } else {
-    cookies.set(REDIRECT_OAUTH_PARAMS_NAME, tokenParamsStr, null, sdk.options.cookies);
-  }
+  // Add oauth_params to both cookies and sessionStorage for broader support
+  cookies.set(REDIRECT_OAUTH_PARAMS_NAME, tokenParamsStr, null, sdk.options.cookies);
+  browserStorage.getSessionStorage().setItem(REDIRECT_OAUTH_PARAMS_NAME, tokenParamsStr);
```

Comment:
do we still need to check for session storage support (will it throw?)

Reply:
I saw the code in 3.x, I think it would be great to use the same logic in both version.